safe_extras/parameters/numeric_parameter.py

```diff
@@ -9,9 +9,6 @@
 class NumericParameter(GenericParameter):
     """A subclass of generic parameter that accepts float or integer only.
 
-    Known issue : You should set the minimum and the maximum value explicitly.
-    See https://github.com/inasafe/inasafe/issues/2468#issuecomment-171124507
-
```

Comment:
There was a discussion about that, at the end we decided to not use git submodule or subtree. But, I can't remember what's the reason. Perhaps @timlinux remember about it?

Anyway, for short solution I will merge the update from InaSAFE repo to the parameter repository.